JitsiConference.js

```diff
@@ -407,17 +407,21 @@ JitsiConference.prototype._init = function(options = {}) {
     this.room.addListener(XMPPEvents.SOURCE_ADD_ERROR, this._removeLocalSourceOnReject);
     this.room.addListener(XMPPEvents.SOURCE_REMOVE, this._updateRoomPresence);
 
-    this.e2eping = new E2ePing(
-        this,
-        config,
-        (message, to) => {
-            try {
-                this.sendMessage(
-                    message, to, true /* sendThroughVideobridge */);
-            } catch (error) {
-                logger.warn('Failed to send E2E ping request or response.', error && error.msg);
-            }
-        });
+    if (config.e2eping && config.e2eping.enabled === true) {
+        this.e2eping = new E2ePing(
+            this,
+            config,
+            (message, to) => {
+                try {
+                    this.sendMessage(
+                        message, to, true /* sendThroughVideobridge */);
+                } catch (error) {
+                    logger.warn(
```

Comment:
nit: eslint max-len is set to 120, there is no need to break into a new line here and there are few other instances in this PR. We are trying to adjust any code we touch to conform to 120 max-len limit.